alpaca_trade_api/entity_v2.py

```diff
@@ -102,5 +103,35 @@ def __init__(self, raw):
 class QuoteV2(Remapped, _NanoTimestamped, Entity):
     _tskeys = ('t',)
 
-    def __init__(self,raw):
+    def __init__(self, raw):
         super().__init__(quote_mapping_v2, raw)
+
+
+class BarV2(Remapped, _NanoTimestamped, Entity):
+    _tskeys = ('t',)
+
+    def __init__(self, raw):
+        super().__init__(bar_mapping_v2, raw)
+
+
+class SnapshotV2:
+
```

Comment:
Did flake8 create these empty lines (in `SnapshotV2` and `SnapshotsV2`)

Reply:
No, removed them